identity/iframe-info-storage-access/index.js

```diff
@@ -0,0 +1,100 @@
+// This will be a list of pending requests that we've sent to the DeSo Identity.
+const req = {};
+let storageAccess = false;
+
+// Vanilla Javascript implementation of UUID v4
+// https://en.wikipedia.org/wiki/Universally_unique_identifier#Version_4_(random)
+// If you're using node.js, you can also use the uuid npm package
+// https://www.npmjs.com/package/uuid
+function uuid() {
+    return 'xxxxxxxx-xxxx-4xxx-yxxx-xxxxxxxxxxxx'.replace(/[xy]/g, function(c) {
+        var r = Math.random() * 16 | 0, v = c == 'x' ? r : (r & 0x3 | 0x8);
+        return v.toString(16);
+    });
+}
+
+// We will use this method as middleware to ensure that the Identity iframe has access to storage.
+function hasStorageAccess () {
+    // The promise will resolve when storage was granted, otherwise it will reject.
+    return new Promise( (resolve, reject) => {
+        // If storage was already granted in the past, we resolve immediately.
+        if (storageAccess){
+            resolve();
+        } else {
+            // We send an info request to the DeSo Identity window to check if we have storage access.
+            const id = uuid();
+            document.getElementById("identity").contentWindow.postMessage({
```

Comment:
Abstract this out and show how to send a sign/jwt/etc?

Reply:
Good point, I think it might be simpler to just add a link to the docs where I'm explaining all the other API calls. Will do that once we merge the new docs.